boards/arm64/xenvm/xenvm-qemu.dts

```diff
@@ -82,4 +83,45 @@
 		compatible = "xen,uart_hvc";
 		status = "okay";
 	};
```

Comment:
Space before new node

Reply:
I still suggest to use overlays instead of changing dts. This is the way it was done in zephyr/samples/subsys/fs/littlefs. Why shouldn't we keep things this way?